CHANGELOG.md

```diff
@@ -4,20 +4,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## 3.0.0-beta
 
 ### Added
-- Protocol v2 support
-- Loggers can now be created from the log package.
-- Already created loggers can be configured from the log package.
-- Updated github.com/sirupsen/logrus dependency to v1.*
-- Add methods for creating events
-### Changed
-- **Breaking change**: Update `Event` type
-- **Breaking change**: Update `Integration` struct type
-- **Breaking change**: package `sdk` now divided into `v1` and `v2`
-### Fixed
-- github.com/Sirupsen/logrus is now github.com/sirupsen/logrus since v1.0.0.
+- Protocol v2 (remote entities) support
+- Concurrency support (via `Synchronized` *option*)
+- Configurable `Logger`
+- Configurable output `Writer`
+- Configurable `Storer`
+- New packages `inventory`, `event` with proper constructors 
+- Integration parametrized creation via optional `Option`s
+### Changes *(breaking changes)*
```

Comment:
Missing the `### Removed` section. We removed support for `v1` right?

Reply:
Indeed, added

Reply:
Sorry But I forgot to mention that you must add a blank line between the previous list and this title. Some Markdown editors does not render this properly and mix them. You know, markdown stuff...

![screen shot 2018-04-30 at 12 44 17 pm](https://user-images.githubusercontent.com/1083296/39423864-3de66198-4c74-11e8-9031-e79e7baa782b.png)